prac_05/hex_colours.py

```diff
@@ -0,0 +1,14 @@
+"""Look up hexadecimal colour codes"""
+
+COLOUR_NAME_TO_CODE = {'AliceBlue': '#f0f8ff', 'Amethyst': '#9966cc', 'AntiqueWhite': '#faebd7',
+                       'Alizarin crimson': '#e32636',
+                       'Apricot': '#fbceb1', 'Baby Pink': '#f4c2c2', 'Banana Yellow': '#ffe135',
+                       'Bittersweet Shimmer': '#bf4f51'}
+
+colour_name = input("Enter name of colour: ")
```

Comment:
If someone enters "baby pink" does it still work? If not, try adding .title() to this line (and the last line).